### DISCUSSION TITLE
Welcome to mMass Discussions!

### DISCUSSION THREAD

**Discussion by mikehollas123** [mikehollas123/mMass#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gyellen** [comment]
I'm a biologist and an experienced programmer, though I've never worked before with python and have no experience with shared repos.  

I've been working on updating mMass to run in more modern versions - python 3.9, newest releases of wxPython, etc.  

I have a working version in **gyellen/mMass** that people are welcome to test out (since I don't know how to make a proper pull request or how to get it to interested parties).  I have recompiled the calculations.c code into both .pyd (for windows) and .so (for macos).

Because my lab is working with small metabolites, and I'm only a novice at the mass spec stuff myself, I have tested the basic functions of mMass, but none of the proteomics functions.  

Where I could certainly use some help is in creating a distributable .exe file for Windows.  I have been unable to get py2exe or pyinstaller to do this.

gary yellen
